docs/featurizers.rst

```diff
@@ -223,12 +217,6 @@ BPSymmetryFunctionInput
 .. autoclass:: deepchem.feat.BPSymmetryFunctionInput
   :members:
 
-OneHotFeaturizer
-----------------
-
-.. autoclass:: deepchem.feat.OneHotFeaturizer
-  :members:
-
 RawFeaturizer
 -------------
```

Comment:
Can you add docs for `Mol2VecFeaturizer` and `MordredFeaturizer` here?

Reply:
I fixed.